source/drive.js

```diff
@@ -0,0 +1,28 @@
+import _curry2 from './internal/_curry2';
+import reduce from './reduce';
```

Comment:
Given that we aren't relying on the currying provided by our public reduce function, we should use `./internal/_reduce` here instead of the public version.